packages/editor/src/components/widgets/code-editor/useCodeEditor.ts

```diff
@@ -14,6 +15,7 @@ export type ModifyAction = (value: string) => string;
 
 export const useMonacoEditor = (options?: { modifyAction?: ModifyAction; scriptAreaDatatypeEditor?: boolean }) => {
   const [editor, setEditor] = useState<monaco.editor.IStandaloneCodeEditor>();
+  const ivyTypes = useMeta('meta/scripting/ivyTypes', undefined, []).data;
```

Comment:
I don't like this!
Now everywhere this hook is used, this request is fired to the backend, even if you never use it...
If we make such conditions, make them based on the location or add more information besides the type to the modifyEditor call.